### DISCUSSION TITLE
Configuration file environment variable

### DISCUSSION THREAD

**Discussion by DaveanandMannie** [lsd-rs/lsd#1094] [Ideas]
Similar to another problem where lsd looks at `LS_COLORS` for colours, the config var could be `LS_CONFIG` or `LSD_CONFIG`
Super selfish, but  being able to keep all my configs in my nvim repo just feels ~nice~

Edit:
I'm on Windows using Wezterm

**Comment by zwpaper** [comment]
Hi, lsd use xdg config standard, you could change it by XDG envs, please check https://github.com/lsd-rs/lsd#configuration

**Reply by DaveanandMannie** [reply]
If I'm understanding the docs correctly. I can create the `XDG_CONFIG_HOME` var to lets say 
C:\users\myuser\AppData\Local\nvim  and it would search in in that dir for LSD/config.yml
And that would work on Windows?. I was under the impression this standard is only honoured on Linux.
Due to XDG being a widely adopted standard, do you think it would affect other programs that use the windows "known folders" system
![image](https://github.com/user-attachments/assets/2ed4dc20-776c-4d10-8fe2-b55dc1230aa6)
As of right now my system does not have XDG envs set

**Comment by DaveanandMannie** [comment]
## If anyone would like to have a non-standard config file location and you are using PowerShell and not set XDG vars user or system-wide.
In your PowerShell profile `$PROFILE` create a small function calling lsd with `--config-file`
```
function lsd {
    <Path to lsd.exe> --config-file <path to config.yaml> @Args
}
```

**Reply by DaveanandMannie** [reply]
I have been less than smart.  I thought sym links where only Linux. I just  made a `junction` with MKLINK /J and it works better than the ps1 func
